lib/x11test.pm

```diff
@@ -1137,4 +1138,98 @@ sub cleanup_firefox_print {
     send_key 'ctrl-d';
     assert_screen 'generic-desktop';
 }
+
+# MM network check: try to ping the gateway, the client and the internet
+sub ensure_client_reachable {
```

Comment:
I have some doubts that this is useful for a production test run and think this is just something that has been used while debugging.